osu.Game/Skinning/SkinManager.cs

```diff
@@ -142,48 +142,42 @@ public void SelectRandomSkin()
         /// </summary>
         public void EnsureMutableSkin()
         {
-            if (CurrentSkinInfo.Value.ID >= 1) return;
-
-            var skin = CurrentSkin.Value;
-
-            // if the user is attempting to save one of the default skin implementations, create a copy first.
-            CurrentSkinInfo.Value = skinModelManager.Import(new SkinInfo
+            CurrentSkinInfo.Value.PerformRead(s =>
             {
-                Name = skin.SkinInfo.Name + @" (modified)",
-                Creator = skin.SkinInfo.Creator,
-                InstantiationInfo = skin.SkinInfo.InstantiationInfo,
-            }).Result.Value;
+                if (s.IsManaged)
+                    return;
```

Comment:
This similarly should also check `IsProtected`. Right now I am able to modify the default skins (without a copy being created) because they actually reside in the same table now rather than being appended outside-of-database like it used to be the case with EF, and so the default skins actually *are* managed.